src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -83,7 +83,7 @@ internal static class IDEDiagnosticIds
 
         public const string PreferBuiltInOrFrameworkTypeDiagnosticId = "IDE0049";
 
-        public const string ConvertAnonymousTypeToTupleDiagnosticId = "IDE0050";
+        // public const string ConvertAnonymousTypeToTupleDiagnosticId = "IDE0050";
```

Comment:
I don't know if we've doc'ed this anywhere.   But we should likely have a PR to remove those docs as well.  It doesn't make sense to have a style forcing people to convert their anon-types to tuples.